aiidalab_widgets_base/structures.py

```diff
@@ -97,7 +111,7 @@ def __init__(
             self.viewer = viewer
         else:
             self.viewer = StructureDataViewer(**kwargs)
-        dlink((self, "structure_node"), (self.viewer, "structure"))
+        link((self, "structure"), (self.viewer, "structure"))
```

Comment:
Please move back to `dlink` and test that things work.

Reply:
done